src/Aspire.Hosting.Azure.ServiceBus/ServiceBusEmulatorContainerImageTags.cs

```diff
@@ -15,11 +15,11 @@ internal static class ServiceBusEmulatorContainerImageTags
     public const string Tag = "1.1.2";
 
     /// <remarks>mcr.microsoft.com</remarks>
-    public const string AzureSqlEdgeRegistry = "mcr.microsoft.com";
+    public const string SqlServerRegistry = "mcr.microsoft.com";
 
-    /// <remarks>azure-sql-edge</remarks>
-    public const string AzureSqlEdgeImage = "azure-sql-edge";
+    /// <remarks>mssql/server</remarks>
+    public const string SqlServerImage = "mssql/server";
 
     /// <remarks>latest</remarks>
-    public const string AzureSqlEdgeTag = "latest";
+    public const string SqlServerTag = "latest";
```

Comment:
I think we should pin this to a version, like we do with the real SqlServer hosting integration:

https://github.com/dotnet/aspire/blob/ef3752be87d556f9673d3a44557bded08c7ad121/src/Aspire.Hosting.SqlServer/SqlServerContainerImageTags.cs#L15

This way we will be less likely to get breaking changes.